.github/workflows/feature-build-hniesb.yml

```diff
@@ -1,36 +1,154 @@
+      - name: Maven run for common code
         run: |
-          echo Add other actions to build,
-          echo test, and deploy your project.
+          cd ${{github.workspace}}/hn-common
+          mvn install
+
+      # Run the maven install for hnsesb code
+      - name: Maven run for hnsecure code
+        run: |
+          cd ${{github.workspace}}/hnsecure
+          mvn install -Dimpl-version=${{env.FEATUREBUILD_NAME}} -Dgit-commit=${GITHUB_SHA::12}
+
+      # Build from docker file
+      - name: Build hnsesb image from Dockerfile
+        id: build-docker-hniesb
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.IMAGE_NAME }}
+          tags: ${{ env.IMAGE_TAGS }}
+          context: ./hnsecure
+          dockerfiles: |
+            ./hnsecure/Dockerfile
+
+      # Upload the images in image repository in tools namespace
+      - name: Push image to registry
+        id: push-image
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ steps.build-docker-hniesb.outputs.image }}
+          tags: ${{ steps.build-docker-hniesb.outputs.tags }}
+          registry: ${{env.IMAGE_REGISTRY}}/${{env.DEFAULT_NAMESPACE}}
+          username: ${{env.IMAGE_REGISTRY_USER}}
+          password: ${{env.OPENSHIFT_TOKEN}}
+ 
+  
+  # This job updates the Gitops repository with image created in  this workflow. 
+  # Updating the image in Deployment.yml will result in deploying that image in OpenShift
+  Deploy:
+    runs-on: ubuntu-latest
+    needs: [createHniEsbFeatureBuild]
+    
+    steps:
+      - name: Step to move image to dev namespace
+        run: |
+          echo "This step should move the image to repository to test namespace"
+
+      # 1. Login to OpenShift
+      - name: Log in to OpenShift
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ env.OPENSHIFT_SERVER }}
+          openshift_token: ${{ env.OPENSHIFT_TOKEN }}
+          insecure_skip_tls_verify: true
+          namespace: ${{ env.DEFAULT_NAMESPACE }}
+
+      # 4. Push the image to selected repository
+      - name: Push Image to dev 
+        run: |
+          oc tag ${{env.DEFAULT_NAMESPACE}}/${{env.IMAGE_NAME}}@${{needs.createHniEsbFeatureBuild.outputs.OPENSHIFT_IMAGE_SHA}} ${{env.DEV_NAMESPACE}}/${{env.IMAGE_NAME}}:${{needs.createHniEsbFeatureBuild.outputs.FEATUREBUILD_NAME}}
+          
+      
+      - name: Prepare deploy key
+        uses: webfactory/ssh-agent@v0.5.3
+        with:
+          ssh-private-key: ${{ secrets.GITOPS_C5839F_PRIVATE_KEY }}
+
+      - name: Check out manifest repo
+        uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.GITOPS_C5839F_PRIVATE_KEY }}
+          repository: bcgov-c/tenant-gitops-c5839f
+          ref: main
+      
+
+      - name: Set up Kustomize
+        uses: imranismail/setup-kustomize@v1
+
+      - name: Update image ID and commit change
+        if: ${{success()}}
```

Comment:
Inconsistent spacing on braces compared to previous lines.

Reply:
updated